lib/attribute-parser.js

```diff
@@ -18,6 +20,9 @@ module.exports = class AttributeParser {
       /^\s+/, // whitespace
       /^[^\s]+/ // word
     ]
+
+    // extend global variables
+    global = Object.assign(global, globalContext)
```

Comment:
See my edit below no need `new` if I understand the exemple right :)
If not to make eslint happy just change `vm` to `VM` it will work